### PR TITLE
Adds GraphQL SSE support to StrawberryShake

### PR DESCRIPTION
**Pull request by PascalSenn**
Adds GraphQL SSE support to StrawberryShake

### CODE REVIEW

File: src/HotChocolate/AspNetCore/src/Transport.Abstractions/Serialization/Utf8JsonWriterHelper.cs [hunk 1]
```diff
@@ -206,14 +206,7 @@ public static void WriteOperationRequest(Utf8JsonWriter writer, OperationRequest
...
         {
-            var element = list[i];
-
-            if (element is null)
-            {
-                continue;
-            }
-
-            WriteFieldValue(writer, element);
+            WriteFieldValue(writer, list[i]);
```
Comment (PascalSenn) [inline src/HotChocolate/AspNetCore/src/Transport.Abstractions/Serialization/Utf8JsonWriterHelper.cs:209]: We need the null values also in arrays

Comment (michaelstaib) [reply src/HotChocolate/AspNetCore/src/Transport.Abstractions/Serialization/Utf8JsonWriterHelper.cs:209]: Ah ... i know why i did that ... if null i wanted to short-circur

writer.WriteNullValue()

in the null check

Comment (michaelstaib) [reply src/HotChocolate/AspNetCore/src/Transport.Abstractions/Serialization/Utf8JsonWriterHelper.cs:209]: uhh ... there are a couple of bugs in there .... I will need to do a separate PR

Comment (michaelstaib) [reply src/HotChocolate/AspNetCore/src/Transport.Abstractions/Serialization/Utf8JsonWriterHelper.cs:209]: damn it michael ;)

Comment (PascalSenn) [reply src/HotChocolate/AspNetCore/src/Transport.Abstractions/Serialization/Utf8JsonWriterHelper.cs:209]: in the Utf8JsonWriter?

**Review by michaelstaib — APPROVED** [review]

**Pull request merged**